.editorconfig

```diff
@@ -0,0 +1,18 @@
+# top-most EditorConfig file
```

Comment:
please do not push this file

Reply:
and also `.gitkeep`

Reply:
.gitkeep is fine, why `.editorconfig` - helps accidental formats done on the code and we can maintain the standard formatting.. now adding to .gitkeep but IMHO worth having it